src/models/network/NetworkBaseController.js

```diff
@@ -9,42 +8,31 @@ export default class NetworkBaseController {
     this.localController = localController;
     this.txParams = txParams;
     this.network = network;
-    this.networkFileName = networkFileName || localController.packageFileName.replace(/\.json\s*$/, `.${network}.json`);
-    if (this.networkFileName === localController.packageFileName) {
-      throw Error(`Cannot create network file name from ${localController.packageFileName}`)
-    }
+    this.networkFile = localController.packageFile.networkFile(network, networkFileName)
   }
 
-  get isDeployed() {
-    throw Error("Unimplemented function isDeployed()");
+  get packageFile() {
+    return this.localController.packageFile;
   }
 
   get packageAddress() {
-    return this.networkPackage.package && this.networkPackage.package.address;
-  }
-
-  get packageData() {
-    return this.localController.packageData;
+    return this.networkFile.packageAddress
   }
 
-  get networkPackage() {
-    if (!this._networkPackage) {
-      this._networkPackage = fs.parseJsonIfExists(this.networkFileName) || _.cloneDeep(this.defaultNetworkPackage);
-    }
-    return this._networkPackage;
+  get isDeployed() {
+    throw Error("Unimplemented function isDeployed()");
   }
 
-  get defaultNetworkPackage() {
-    return { contracts: {} };
+  isLib() {
```

Comment:
Same as above. `isDeployed` is a getter but `isLib` a function, we should be consistent.